packages/ResponsesAccordion/src/components/Responses/Responses.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import responsesStyles from "./Responses.styles";
+
+const responseShape = {
+  heading: PropTypes.node.isRequired,
+  body: PropTypes.node,
+};
+
+const propTypes = {
+  responses: PropTypes.arrayOf(PropTypes.shape(responseShape)),
+};
+
+const defaultProps = {
+  responses: [],
+};
+
+const Response = ({ heading, body }) => {
+  const I18n = useI18n();
+
+  return (
+    <>
+      <dt>{heading}</dt>
+      <dd>{body || <em>{I18n.t("responsesAccordion.no_response")}</em>}</dd>
+    </>
+  );
+};
+
+Response.propTypes = responseShape;
+Response.defaultProps = {
+  body: null,
+};
+
+const Responses = ({ responses }) => {
+  return (
+    <dl css={responsesStyles}>
+      {responses.map(({ heading, body }) => (
+        <Response heading={heading} body={body} key={heading} />
```

Comment:
Another case of a node being used for key

Reply:
Yeah, and probably shouldn't rely on it's uniqueness either, *sigh*.  These are static, so I feel like I should just be able to use the `index`, but I'm afraid of the eslint / zdev cops 👮‍♂️

Reply:
you can use an index as long the component is quite static (will not rerender), and seem to me this component is unlikely to rerender the headings or even the component, seems more a presentational component than an input component. So should be fine to use indexes.

Reply:
@nahumzs see https://github.com/acl-services/paprika/pull/125#discussion_r305509234
(It will definitely re-render btw; we're not doing anything to prevent that)